.github/workflows/bindings.yml

```diff
@@ -45,6 +45,10 @@ jobs:
         uses: maxim-lobanov/setup-xcode@v1
         with:
             xcode-version: '16.2.0'
+      - name: swift format
+        run: |
+            cd crypto-ffi/bindings/swift/WireCoreCrypto
+            swift format lint -r .
```

Comment:
@coriolinus @typfel 
Without `-s`, this will only emit warnings, but will not actually exit with a non-zero exit code.
Don't we want to fail the CI if the files are not properly formatted?